### PR TITLE
pd-control: add new config of balance-hot-region-scheduler

### PR DESCRIPTION
**Pull request by HunDunDM**
Signed-off-by: HunDunDM <hundundm@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

pd-control: add new config of balance-hot-region-scheduler

Related features are GA at v5.2.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
  - Features: [PD#3869](https://github.com/tikv/pd/issues/3869) [PD#3900](https://github.com/tikv/pd/pull/3900)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1050,7 +1072,25 @@ Encoding 格式示例：
 - `src-tolerance-ratio` 和 `dst-tolerance-ratio` 是期望调度器的配置项。`tolerance-ratio` 的值越小，调度就越容易。当出现冗余调度时，你可以适当调大这个值。
 
     ```bash
-    >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.05
+    >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
+    ```
+
+- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群未全部升级到 v5.2 及以上版本时，它们不会生效。通常用户不需要修改这些配置项。
```
Comment (lhy1024) [inline pd-control.md:1078]:
```suggestion
- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群未全部升级到 v5.2 及以上版本时，它们不会生效，使用兼容配置。通常用户不需要修改这些配置项。
```

File: pd-control.md [hunk 2]
```diff
@@ -1050,7 +1072,25 @@ Encoding 格式示例：
...
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set read-priorities query,byte
+    ```
+
+- `strict-picking-store` 是控制热点调度搜索空间的开关，打开时会在保证稳定性的前提下进行热点调度。通常为打开，关闭可能会导致第二维度的均衡度降低。通常用户不需要修改这个配置项。
```
Comment (lhy1024) [inline pd-control.md:1084]:
```suggestion
- `strict-picking-store` 是控制热点调度搜索空间的开关，打开时会在保证稳定性的前提下进行热点调度。通常为打开，关闭后只能保证第一优先级维度的均衡性，可能会导致其他维度的均衡度降低。通常用户不需要修改这个配置项。
```

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

File: pd-control.md (hunk 1 above)
Comment (TomShawn) [inline pd-control.md:1078]:
```suggestion
- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效。通常用户不需要修改这些配置项。
```

File: pd-control.md [hunk 3]
```diff
@@ -1050,7 +1072,25 @@ Encoding 格式示例：
...
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set strict-picking-store true
+    ```
+
+- `enable-for-tiflash` 是控制热点调度是否对 TiFlash 生效的开关。通常为打开，关闭后将不会产生 TiFlash 之间的热点调度。
```
Comment (TomShawn) [inline pd-control.md:1090]:
```suggestion
- `enable-for-tiflash` 是控制热点调度是否对 TiFlash 生效的开关。通常为打开，关闭后将不会产生 TiFlash 副本之间的热点调度。
```

Comment (HunDunDM) [reply pd-control.md:1090]: Can it be changed to "TiFlash 实例"?

Comment (TomShawn) [reply pd-control.md]: Done

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**